js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -1380,23 +1380,23 @@ titleUsers=Users
 whoWillAppearLinkText=Who will appear in this group list?
 ldapFullNameAttribute=LDAP full name attribute
 createClientError=Could not create client\: '{{error}}'
-deleteConfirmClientScopes=Are you sure you want to delete this client scope
+deleteConfirmClientScopes=Are you sure you want to delete this client scope?
 forceAuthenticationHelp=Indicates whether the identity provider must authenticate the presenter directly rather than rely on a previous security context.
 testClusterAvailability=Test cluster availability
 reGenerateSigning=Regenerate signing key for this client
 authorizationEncryptedResponseAlgHelp=JWA Algorithm used for key management in encrypting the authorization response when the response mode is jwt. This option is needed if you want encrypted authorization response. If left empty, the authorization response is just signed, but not encrypted.
-deleteConfirmGroup_other=Are you sure you want to delete these groups.
+deleteConfirmGroup_other=Are you sure you want to delete these groups?
 scopePermissions.users.manage-description=Policies that decide if an administrator can manage all users in the realm
 defaultACRValuesHelp=Default values to be used as voluntary ACR in case that there is no explicit ACR requested by 'claims' or 'acr_values' parameter in the OIDC request.
 minimumACRValueHelp=Minimum ACR to be enforced by Keycloak. Overrides lower ACRs explicitly requested by 'acr_values' or 'claims', unless they are marked as essential.
 membershipAttributeType=Membership attribute type
 eventTypes.PUSHED_AUTHORIZATION_REQUEST.name=Pushed authorization request
-included.client.audience.tooltip=The Client ID of the specified audience client will be included in audience (aud) field of the token. If there are existing audiences in the token, the specified value is just added to them. It won't override existing audiences.
+included.client.audience.tooltip=The Client ID of the specified audience client will be included in the audience (aud) field of the token. If there are existing audiences in the token, the specified value is just added to them. It won't override existing audiences.
```

Comment:
```suggestion
included.client.audience.tooltip=The Client ID of the specified audience client will be included in the audience (aud) field of the token. If the token includes audiences, the specified value is added to them. It will not override existing audiences.
```